Bender.yml

```diff
@@ -71,6 +71,7 @@ sources:
 
   - target: simulation
     files:
+      - src/axi_dumper.sv
```

Comment:
We will have to update the other two dependency tools as well. I can do that in the release commit though.